### DISCUSSION TITLE
Quick fix - Auto-Import for unambiguous class

### DISCUSSION THREAD

**Discussion by Adolio** [BowlerHatLLC/vscode-as3mxml#730] [Q&A] [answered]
Hi there,

I'm wandering if it's possible to enable auto-import when a type is obvious.

![image](https://github.com/BowlerHatLLC/vscode-as3mxml/assets/1126990/dfe6fd59-d3c2-4c49-8919-3da7245654a0)

I tried the option `"as3mxml.sources.organizeImports.addMissingImports": true` in my `settings.json` but it doesn't seem to do anything automatically.

Any thoughts? Thanks in advance!

**Comment by joshtynjala** [comment, marked as answer]
The “Organize Imports” command will auto-import unambiguous classes. That’s what the setting is meant to control.

Otherwise, imports are automatically added only when you select a class from the completion list. As in your screenshot, you use the quick fix to manually add an import if you typed the class name without completion.
